docs/user_guide/method/method.rst

```diff
@@ -0,0 +1,2384 @@
+**mrsimulator**, all simulations are performed in the high-field limit and
+further assume that all spin-spin couplings are in the weak limit.
+
+To write a custom Method in *mrsimulator*, you'll need to determine the desired
+transition pathways and select the desired transitions during each
+**SpectralEvent** or **DelayEvent**. Keep in mind, however, that **Method**
+objects are designed without any details of the spin systems upon which they
+will act. For example, in the density matrix of a spin system ensemble, one
+could easily identify a transition by its row and column indexes. However, those
+indexes depend on the spin system and how the spins and their eigenstates have
+been assigned to those indexes. Instead, we need a spin-system agnostic approach
+for selecting transitions.
+
+
+
+Spin Transition Symmetry Functions
+''''''''''''''''''''''''''''''''''
+
+One way you can select a subset of single-spin transitions if you don't
+know the energy eigenstate quantum numbers is to request all transitions whose
+single-spin transition symmetry function, :math:`\text{p}_I` symmetry function
+is :math:`-1`, i.e.,
+
+.. math::
+    \text{p}_I(m_f,m_i) = m_f - m_i = -1.
+
+The :math:`\text{p}_I` single-spin transition symmetry function is also known as
+the single-spin `coherence order of the transition
+<https://doi.org/10.1016/0022-2364(84)90142-2>`_.
+
+.. note::
+
+    In the high field limit, only single-spin transitions with
+    :math:`{\text{p}_I = \pm 1}` are directly observed.  Since the
+    evolution frequencies of the :math:`\ketbra{j}{i}` and
+    :math:`\ketbra{i}{j}` transitions are equal in magnitude but opposite
+    in sign, the convention is to only present the :math:`{\text{p}_I = - 1}``
+    transition resonances in single-quantum spectra.
+
+By selecting only single-spin transitions with :math:`\text{p}_I = -1`, you get
+all the "observed" transitions from the set of all possible transitions.
+Similarly, you can use  :math:`\text{p}_I` to select any subset of single-spin
+transitions, such as double-quantum :math:`(\text{p}_I = \pm 2)` transitions,
+triple-quantum :math:`(\text{p}_I = \pm 3)` transitions, etc.
+
+While specifying :math:`\text{p}_I` alone is not enough to select an individual
+single-spin transition, any individual single-spin transition can be
+identified by a combination of :math:`\text{p}_I` and the single-spin
+transition symmetry function :math:`\text{d}_I`, given by
+
+.. math::
+
+    \text{d}_I(m_i,m_j) =  ~m_j^2 - m_i^2.
+
+You can verify this from the values of :math:`\text
+{p}_I` and :math:`\text{d}_I` for all single-spin transitions
+for :math:`I=1`, :math:`I=3/2` and :math:`I=5/2` shown below.  Note
+that :math:`\text{d}_I = 0` for all transitions in a :math:`I=1/2` nucleus.
+
+
+.. figure:: ../../_static/SpinOneThreeHalves.*
+    :width: 500
+    :alt: figure
+    :align: center
+
+
+.. figure:: ../../_static/SpinFiveHalf.*
+    :width: 550
+    :alt: figure
+    :align: center
+
+
+.. note::
```

Comment:
Here's the other long note. Should be own sub-section

Reply:
I put it in a note because the reader could skip past this.   Seems like the only options are note and warning.   Is there some for some kind of "sidebar"?   https://lindasclare.com/2022/02/writing-call-outs-sidebars-and-breakout-boxes/

Reply:
Created dropdown using html